administration/src/cards/CardBlueprint.ts

```diff
@@ -54,7 +54,7 @@ export class CardBlueprint {
 
   isFullNameValid(): boolean {
     const encodedName = new TextEncoder().encode(this.fullName)
-    return this.fullName.length > 0 && encodedName.length < MAX_NAME_LENGTH
+    return this.fullName.length > 0 && encodedName.length <= MAX_NAME_LENGTH
```

Comment:
We also add this as a db constraint. Or is it already too late for that?

Reply:
```js
A check constraint does check the values already in the table and cannot be created if there are rows violating the condition. That's the point of a constraint, assert that the data meets the condition.
```

Reply:
hmm okay. Then I guess we have to leave it as it is. Otherwise we'd need to remove/change entries in the db.